### ISSUE TITLE
How to get private key BitPay and import to bitcoin-cli?

### ISSUE THREAD

**Issue by botaniQQQ** [bitpay/copay#8797]
Hello.

I use BitPay 
![jvvxxjv](https://user-images.githubusercontent.com/3118010/41626878-9775427c-7426-11e8-90bc-28694d0e9767.png)

and have many addresses
![7bkeqmz](https://user-images.githubusercontent.com/3118010/41626884-9eb42bac-7426-11e8-8a23-dbfc26eae05f.png)

but with balance only address **1A...**

I have xprv: xprv9s21Zr...

I generate Private Key (WIF) using https://bip32jp.github.io/english/
![mdilppf](https://user-images.githubusercontent.com/3118010/41627149-c37cfbf2-7427-11e8-814f-9f51145aa488.png)

```
~# bitcoin-cli importprivkey "Private Key (WIF)" "" flase

~# bitcoin-cli getaddressesbyaccount ""
[
  "1217...",
  ...
]

~# bitcoin-cli getbalance
0.00000000
```

How to import my address **1A...**?

**Comment by BGalliotti** [comment]
Hello @botaniQQQ ,

Thanks for reaching out to us.

We do not understand what you exactly want to do. 
You have a wallet, it has many addresses associated with it, and of them has funds.
You want to import that address, right?
You do not import addresses, you import wallets.

So, you import the whole wallet, it will include the addresses and funds and you can you use them.

Please let us know if we are on the same page and if you have any additional questions.
Kind regards.

**Comment by botaniQQQ** [comment]
>  You want to import that address, right?
Yes, how to import an address **1A...** to use it in Bitcoin Core (command line)?
